bioio/plugins.py

```diff
@@ -222,6 +225,14 @@ def get_plugins(use_cache: bool) -> Dict[str, List[PluginEntry]]:
             # Add plugin entry
             plugin_entry = PluginEntry(plugin, reader_meta, timestamp)
             for ext in plugin_entry.metadata.get_supported_extensions():
+                # Always remove leading "." if there is one
+                # Some plugin authors may add it, but it's not necessary
+                # And worse can cause this plugin list to have multiple entries
+                # for the same extension (e.g. ".tif" and "tif")
+                if ext.startswith("."):
```

Comment:
I can't remember if our base Reader docs suggest that extension strings start with dot or not, but this change seems good.

Reply:
I would like to make even more changes to this but happy to do so in another PR: https://github.com/bioio-devs/bioio/pull/55/files#r1688696186